src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +49,43 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        return (dateParams.length != 0 && dateParams[0] > 1970
```

Comment:
simplify this method: return Optional.of(...) 
and let's use try-catch block here

Reply:
Done